backend/src/main/java/com/woowacourse/moragora/dto/ParticipantResponse.java

```diff
@@ -13,35 +12,28 @@ public class ParticipantResponse {
     private final Long id;
     private final String email;
     private final String nickname;
-    private final String attendanceStatus;
     private final int tardyCount;
+    private final Boolean isMaster;
 
-    private ParticipantResponse(final Long id,
-                                final String email,
-                                final String nickname,
-                                final String attendanceStatus,
-                                final int tardyCount) {
+    @Builder
+    public ParticipantResponse(final Long id,
+                               final String email,
+                               final String nickname,
+                               final int tardyCount,
+                               final boolean isMaster) {
         this.id = id;
         this.email = email;
         this.nickname = nickname;
-        this.attendanceStatus = attendanceStatus.toLowerCase(Locale.ROOT);
         this.tardyCount = tardyCount;
+        this.isMaster = isMaster;
     }
 
-    public ParticipantResponse(final Long id,
-                               final String email,
-                               final String nickname,
-                               final Status attendanceStatus,
-                               final int tardyCount) {
-        this(id, email, nickname, attendanceStatus.name(), tardyCount);
-    }
-
-    public static ParticipantResponse of(final User foundUser, final Status attendanceStatus, final int tardyCount) {
+    public static ParticipantResponse of(final User foundUser, final int tardyCount, final boolean isMaster) {
```

Comment:
```suggestion
    public static ParticipantResponse of(final Participant participant, final int tardyCount)
```
이렇게 바꿔볼 수도 있을 거 같아요!

Reply:
오 이게 더 좋은 방법 같아요!!